src/sqlite_db.cpp

```diff
@@ -277,34 +289,24 @@ SQLitePerfDb::SQLitePerfDb(const std::string& filename_,
 
         // clang-format on
         {
+            const auto lock = shared_lock(lock_file, GetLockTimeout());
+            MIOPEN_VALIDATE_LOCK(lock);
             // clang-format off
             const auto check_tables =
                 "SELECT name FROM sqlite_master "
                 "WHERE "
                   "type = 'table' AND "
-                  "(name = 'config');";
+                  "(name = 'config' OR name = 'perf_db');";
```

Comment:
This masks a bug, if only one of the tables is created.

Reply:
If this fixes the Jenkins issue then you can implement this fix, but leave off the rest of the locking changes.